src/app-env.ts

```diff
@@ -0,0 +1,12 @@
+export class AppEnv {
```

Comment:
As mentioned above, I would inject all of these via webpack.config. We could also have some yaml, json or env file where to read from. For now should be just fine to add them to the `env` object in `webpack-config.js`. They get exposed then to `process.env`.